package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@prosperworks/eslint-config",
+  "name": "eslint-config-copper",
```

Comment:
I had changed the name a few months ago but never ran `npm i` so the package-lock could get the memo.